getting_started/step_by_step/scene_tree.rst

```diff
@@ -140,10 +146,18 @@ another one. The simple way to do this is to use the
 :ref:`SceneTree.change_scene() <class_SceneTree_change_scene>`
 function:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _my_level_was_completed():
-        get_tree().change_scene("res://levels/level2.tscn")
+    get_tree().change_scene("res://levels/level2.tscn")
+
+ .. code-tab:: csharp
+
+    public void _my_level_was_completed()
```

Comment:
I'd recommend to follow PascalCase in C# code snippets, as it's the standard coding convention for the language.

I'm not sure as to how to think about using the `_` prefix though. I understand that it's used to denote a callback method in Godot and there's an on going discussion about the issue (godotengine/godot#14466).